common/system/filtering/filters.cpp

```diff
@@ -380,6 +380,12 @@ static void filterPositionZ(SensorData_s* curSensorVals, bool hasPassedApogee) {
   // TODO check what order these "should" run in. Worst case we're off by one
   // iteration We don't update accelerations till after this, so the z
   // acceleration should still be from the last timestep
+
+  //Updates Kalman filter timestamps to the current loop and uses new vlaues to calculate the current dt
+  //Note: Check that predict() does not run when the timestamp is 0, otherwise it will crash when initalizing
```

Comment:
Does this crash still happen? If so, how?

Reply:
Note from before, fixed this to prevent possible divide by 0 error